lib/rescue_from_unless_local.rb

```diff
@@ -12,7 +12,9 @@ module ClassMethods
     # exceptions (good in development) or rescue exceptions (good in production).  Because
     # the test of the setting is done at the moment the exception is being handled, we
     # can change the value in specs to test both the exception and the rescuing behavior.
-    def rescue_from_unless_local(*klasses, with: nil, &block)
+    # If send_to_sentry is true and the exception is not already going to be reraised,
+    # the exception will be captured off to Sentry.
+    def rescue_from_unless_local(*klasses, with: nil, send_to_sentry: false, &block)
```

Comment:
why is the default for send_to_sentry true for base_controller, but false here?

Reply:
the one in the base controller isn't a default, its explicitly setting it to `true`.  Making the default in the `lib` be false makes this a non-breaking change.